docs/caikit-isvc.yaml

```diff
@@ -9,7 +9,10 @@ metadata:
 spec:
   predictor:
     model:
+      apiVersion: serving.kserve.io/v1alpha2
```

Comment:
do we need to update the KServe CRD as well? currently we ship v1beta1 @Xaenalt

Reply:
I think we want to align this with the current KServe version, so I think we'd want this to be v1beta1